packages/nextjs/app/builders/0x83DD842BB14aacE72A1a68a129D134Ee3EeBbbd4/page.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+import { AcademicCapIcon, XMarkIcon } from "@heroicons/react/24/outline";
+import { MapPinIcon } from "@heroicons/react/24/solid";
+
+const Profile: NextPage = () => {
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10 justify-center">
+        <div className="flex gap-10">
+          <div className="">
+            <Image src="/builders.jpeg" className="rounded-full" alt="Sachin Varma" width={200} height={40} />
```

Comment:
This is a square image, so maybe you mean height={200}? Doesn't seem to affect how it's rendered, though.